tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -183,6 +183,16 @@ cl::opt<bool> SPIRVAllowUnknownIntrinsics(
     cl::desc("Unknown LLVM intrinsics will be translated as external function "
              "calls in SPIR-V"));
 
+static cl::opt<SPIRV::DebugInfoEIS> DebugEIS(
+    "debug-info-eis", cl::desc("Set debug info EIS:"),
```

Comment:
Typically SPIR-V translator options begin with `-spirv`.
"EIS" is not quite obvious abbreviation. Can we use "version" instead?
```suggestion
    "spirv-debug-info-version", cl::desc("Set SPIR-V debug info version:"),
```

Reply:
Fixed